tests/test_entity_mixins.py

```diff
@@ -344,6 +364,70 @@ def test_bad_value_error(self):
         with self.assertRaises(entity_mixins.BadValueError):
             SampleEntityTwo(self.cfg, one_to_many=1)
 
+    def test_eq_none(self):
+        """Test method ``nailgun.entity_mixins.Entity.__eq__`` against None
+
+        Assert that ``__eq__`` returns False when compared to None.
+
+        """
+        alice = SampleEntity(self.cfg, id=1, name='Alice')
+        self.assertNotEquals(alice, None)
```

Comment:
Use `assertNotEqual` (without the `s` at the end) since `assertNotEquals` is deprecated and will produce a warning:

```
In [7]: t.assertNotEquals
Out[7]: <bound method TestCase._deprecate.<locals>.deprecated_func of <__main__.Test testMethod=runTest>>

In [9]: t.assertNotEquals??
Signature: t.assertNotEquals(*args, **kwargs)
Source:   
        def deprecated_func(*args, **kwargs):
            warnings.warn(
                'Please use {0} instead.'.format(original_func.__name__),
                DeprecationWarning, 2)
            return original_func(*args, **kwargs)
File:      /usr/lib64/python3.5/unittest/case.py
Type:      method
```